_posts/2021-04-15-jesse-frost.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Jesse Posner on Flexible Round-Optimized Schnorr Threshold Signatures (FROST)"
+permalink: /blog/2021/04/15/frost/
+layout: post
+author: jesse
+category: "technical"
+description: "Jesse Posner explains the FROST signature scheme."
+---
+
+The promise of Bitcoin is only as secure as the cryptographic keys that
+authenticate its operations: they are the keys to freedom in the digital age.
+However, the requirements for Bitcoin key management are becoming more demanding
+with innovative off-chain protocols such as the [Lightning Network][ln] and
+[Discreet Log Contracts][dlc] that introduce new roles and responsibilities for
+keys. In addition, Bitcoin's relentless rise in value creates a need for
+sophisticated systems that can help make rubber-hose cryptanalysis uneconomical.
+
+Off-chain protocols facilitate smart contracts that are highly scalable and
+private, which enable a vast array of new markets and financial instruments.
+However, many of these protocols present challenges from a key management
+perspective because they rely on “hot” keys that must be readily accessible and
+frequently reused, unlike safer "cold" keys that are rarely accessed and never
+reused. With the introduction of [Taproot][bip341], we can begin to bridge this
+divide between a "hot" and "cold" key with distributed key generation and
+threshold signing.
+
+Distributed key generation uses secure multi-party computation ([MPC][mpc wiki])
+to generate a pair of public and private keys with multiple parties such that no
+party learns the private key. Instead, each party learns a "share" of the
```

Comment:
This sentence could be slightly reformulated to make it clearer. "a pair of public and private keys" is slightly ambigious since it could mean "a pair of key pairs", and "with multiple parties" is separated from the thing it's refering to ("secure multi-party computation") and might be parsed as refering to "private keys".

Perhaps something like:

```suggestion
In distributed key generation, multiple parties use secure multi-party computation ([MPC][mpc wiki])
to generate a single public/private key pair, such that no
party learns the private key. Instead, each party learns a "share" of the
```